src/lib.rs

```diff
@@ -110,18 +112,20 @@ mod test {
 
     #[test]
     fn should_get_config_values_from_env() {
-        set_env("localhost:1234", "", "2", "3");
+        set_env("localhost:1234", "", "2", "3", "");
         let config = config_from_env();
         assert_eq!("localhost:1234".to_string(), config.hosts);
         assert_eq!(30, config.timeout);
         assert_eq!(2, config.wait_before);
         assert_eq!(3, config.wait_after);
+        assert_eq!(1, config.wait_sleep_interval);
```

Comment:
Could you please add a unit test to verify that the wait_sleep_interval is effectively read from a global variable if present?